pkg/pebinaryexecutor/pebinary.go

```diff
@@ -254,6 +273,15 @@ func getArgs(argsMap map[string]string) []string {
 	return args
 }
 
+func argIndex(args []string, s string) (int, error) {
+	for i, arg := range args {
+		if strings.Contains(arg, s) {
+			return i, nil
+		}
+	}
+	return indexNotFound, errors.New("no matching argument found")
```

Comment:
I don't think this needs a constant. The convention is to return `nil` or the type default when err is non-nil. Just do `return 0, errors.New(...)` and drop the constant.

Reply:
@briandowns suggest using a constant. What should I do?

Reply:
The const was for the `-1` since it would generally imply some meaning. If it's just 0, that's fine.

Reply:
@brandond Method removed.